.changeset/little-pillows-fry.md

```diff
@@ -0,0 +1,13 @@
+---
+"@salt-ds/core": minor
+---
+
+`Text` component has two new variants: `Action` and `Notation`. See [Storybook](https://storybook.saltdesignsystem.com/?path=/story/core-text-text-qa--all-variants-grid) for more information.
+
+```tsx
```

Comment:
Are we avoiding mentioning Action here?

Reply:
originally kept it succinct in theme PR but now that it's separated i've added Action too

Reply:
changesets should always cover the complete set of new additions